### DISCUSSION TITLE
`reactRender is not a function` when using Antd with NextJS App Router

### DISCUSSION THREAD

**Discussion by baoanh177** [community/community#147157] [Programming Help] [answered]
### Body

I'm initializing a NextJS AppRouter project with Antd. I've been looking for a solution but can't find it, can anyone help me, pls.
- Here are project dependencies:
![image](https://github.com/user-attachments/assets/bb10590d-fdbb-4893-96cf-f662fb6938d4)

- src/app/layout.tsx
![image](https://github.com/user-attachments/assets/47db6f96-9d55-4ed0-99db-26a813bfc863)

- src/app/page.tsx
![image](https://github.com/user-attachments/assets/cf1437c0-7292-4d37-9d22-3db37f0a2bba)

Everything is fine until I click the button, console.log() inside the handClick function runs but gets an error
![image](https://github.com/user-attachments/assets/00e8d115-91df-4860-9c0a-2c84cc357a62)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by haseeb-ariff** [comment, marked as answer]
The same thing happened to me, so I downgraded the next version to version 14.2.20, which fixed the bug.

**Reply by baoanh177** [reply]
It's works! Thank you
